aspnet/fundamentals/logging.rst

```diff
@@ -3,228 +3,379 @@
+``AddConsole`` and ``AddDebug`` are extension methods on ``ILoggerFactory``. They are provided by the `Microsoft.Extensions.Logging.Console` and `Microsoft.Extensions.Logging.Debug` NuGet packages.  Each extension method calls the ``AddProvider`` method, passing in an instance of the provider.
 
-.. image:: logging/_static/console-logger-output.png
+To call logging methods from one of your classes, get a logger object from DI and store it in a field, as shown in this example.
 
-You may see more than one log statement per web request you make in your browser, since most browsers will make multiple requests (i.e. for the favicon file) when attempting to load a page. Note that the console logger displayed the log level (``info`` in the image above) followed by the category (``[Catchall Endpoint]``), and then the message that was logged.
+.. literalinclude:: logging/sample/src/TodoApi/Controllers/TodoController.cs
+  :language: c#
+  :start-after: snippet_LoggerDI
+  :end-before: endregion
+  :emphasize-lines: 4,7,10
+  :dedent: 4
 
-The call to the log method can utilize a format string with named placeholders (like `{path}`). These placeholders are populated in the order in which they appear by the args values passed into the method call. Some logging providers will store these names along with their mapped values in a dictionary that can later be queried. In the example below, the request path is passed in as a named placeholder:
+You can then create logs by calling methods named ``Log`` + the severity level of the log to write, as in this example.
 
-.. literalinclude:: logging/sample/src/TodoApi/Startup.cs
+.. literalinclude:: logging/sample/src/TodoApi/Controllers/TodoController.cs
   :language: c#
-  :lines: 54
-  :dedent: 16
+  :start-after: snippet_CallLogMethods
+  :end-before: endregion
+  :emphasize-lines: 3,7
+  :dedent: 8
+
+With the sample code shown above, you'll see logs in the console when you run from the command line, and in the Debug window when you run in Visual Studio in Debug mode. 
+
+If you run the `sample application <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/logging/sample>`__ from the command line (``dotnet run``) and go to URL ``http://localhost:5000/api/todo/0``, you see console output similar to the following example:
+
+.. code-block:: none
+
+  info: Microsoft.AspNetCore.Hosting.Internal.WebHost[1]
```

Comment:
@BrennanConroy - Purpose is to show log text that is generated, and for that purpose a text block is readable at all browser widths and on devices.